cypress/elements/catalog/products/assign-products-selectors.js

```diff
@@ -0,0 +1,7 @@
+/* eslint-disable sort-keys */
```

Comment:
Mentioned it ealier - eslint is used to enforce consistent code style, don't block it without good reason.